example/20-componentconfig-gardenlet.yaml

```diff
@@ -114,6 +114,7 @@ featureGates:
   CopyEtcdBackupsDuringControlPlaneMigration: true
   DefaultSeccompProfile: true
   CoreDNSQueryRewriting: true
+  IPv6SingleStack: true
```

Comment:
That's my fault. I missed adding the results of my out-of-band discussion with @rfranzke on github.

The reasoning is that the `IPv6SingleStack` feature gate doesn't change apiserver's/gardenlet's default behavior. It only allows the creation of IPv6 seeds and shoots. The behavior only changes when creating such objects.
We agreed on enabling it by default in the local setup so that devs don't have to modify the configuration to get started with IPv6. If devs don't want to use IPv6, nothing changes.
Hence, I clarified the implications of enabling the feature gate in `feature_gate.md`.